packages/react/src/components/library/card-auth/CardAuth.scss

```diff
@@ -1,18 +1,19 @@
 @use "../../../variables.scss" as *;
 
-.auth-card {
+.content .auth-card {
```

Comment:
Does the parent component have content selector?

Reply:
Yes, you are right, not always. This broke the standalone log in page.
Fixed this and added (returned) a test assertion to check the form in logged-out state.